impala/util.py

```diff
@@ -202,13 +203,19 @@ def get_cookie_expiry(c):
 
 
 def get_all_matching_cookies(cookie_names, path, resp_headers):
+    matching_cookies = None
```

Comment:
Why add this line

Reply:
Thanks, it does nothing, removing in next patch